### PR TITLE
fix: polishing employee page

### PR DESCRIPTION
**Pull request by mikaelbr**
- Makes it load lazily
- Matches design and typography
- Adds proper links
- Changes to be actual grid that supports longer names
- makes it more small-screen friendly

Fixes https://github.com/varianter/variant.no/issues/898

OBS: I haven't checked if the Employee card is used several places and if this change breaks anything outside the listings 😅

Screenshots:

Screenshot 2024-11-28 at 22 39 32
Screenshot 2024-11-28 at 22 39 24

### CODE REVIEW

File: src/components/employeeCard/EmployeeCard.tsx [hunk 1]
```diff
@@ -30,38 +30,58 @@ export default function EmployeeCard({
           <div className={styles.employeeImage}>
             <Image
               src={employee.imageUrl}
-              alt=""
+              alt={employee.name}
               style={{ objectFit: "cover" }}
               fill={true}
             />
           </div>
         </Link>
```
Comment (anemne) [inline src/components/employeeCard/EmployeeCard.tsx:38]: Maybe we should remove the link to the mini-cv for now?

**Review by anemne — APPROVED** [review]
Looks good!

File: src/components/employeeCard/EmployeeCard.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/components/employeeCard/EmployeeCard.tsx:38]: I think it adds value to be able to link to contact information for a particular person 🤔

Comment (anemne) [reply src/components/employeeCard/EmployeeCard.tsx:38]: Agreed that we should have links to contact info, but do we need link to the mini-cv page?

Comment (mikaelbr) [reply src/components/employeeCard/EmployeeCard.tsx:38]: I mean contact page. As in "Contact this person http://localhost:3000/no/folk/mn"

**Pull request merged**